database/src/prepare.ts

```diff
@@ -20,10 +20,10 @@ export default async (): Promise<void> => {
   await con.connect()
 
   // Create Database `gradido_community`
-  await con.query(`
-    CREATE DATABASE IF NOT EXISTS ${CONFIG.DB_DATABASE} 
-      DEFAULT CHARACTER SET utf8mb4
-      DEFAULT COLLATE utf8mb4_unicode_ci;`)
+  await con.query('\
+    CREATE DATABASE IF NOT EXISTS `${CONFIG.DB_DATABASE}`\
```

Comment:
Wasn't this a problem with the previous skeema setup aswell? I copied the query from there?!